client/src/components/features/SeatChooser/SeatChooser.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react';
 import { Button, Progress, Alert } from 'reactstrap';
+import io from 'socket.io-client';
+
 
 import './SeatChooser.scss';
 
 class SeatChooser extends React.Component {
   componentDidMount() {
-    const { loadSeats } = this.props;
+    const { loadSeats, updateSeats } = this.props;
     loadSeats();
-    let intervalId = setInterval(() => loadSeats(), 120000);
-    this.setState({ intervalId: intervalId});
-  }
-  componentWillUnmount(){
-    clearInterval(this.state.intervalId);
+    this.socket = io((process.env.NODE_ENV === 'production') ? 'localhost:3000' : 'localhost:8000');
```

Comment:
zmień url na produkcję. Z tego co napisałeś wynika, że na prodzie sockety mają się odpalać na porcie 3000

Reply:
Czy mógłbyś mi to wytłumaczyć tak łopatologicznie? Poprawiłem, ale nie do końca rozumiem to

Reply:
chodzi o to, że kiedy środowisko będzie produkcyjne, nie serwujesz strony na localhoście, tylko na prawdziwym serwerze, dlatego trzeba podmienić urla. Localhost jest tylko na potrzeby developmentu i jak sama nazwa mówi jest to środowisko lokalne, nikt poza Tobą nie ma dostępu do tego środowiska